mslib/msui/mscolab.py

```diff
@@ -1056,8 +1056,11 @@ def handle_delete_operation(self):
                         self.reload_operations()
                         self.signal_operation_removed.emit(self.active_op_id)
                         logging.debug("activate local")
-                        self.ui.listFlightTracks.setCurrentRow(0)
-                        self.ui.activate_selected_flight_track()
+                        if self.ui.listFlightTracks.count() == 0:
```

Comment:
in mscolab we can remove "all" operations from the list, while we can't in the local flight path (left site of the UI)
When we do this, we have to switch to an active operation, which is done by switch_to_local